main.go

```diff
@@ -34,6 +37,13 @@ func main() {
 
 	setupLogger()
 
+	if *version {
+		if err = printVersion(); err != nil {
```

Comment:
```suggestion
		if err := printVersion(); err != nil {
```

Reply:
the error var is declared before this, so this wouldn't be required. But I changed it.

Reply:
I prefer to not shadow anything (usually `err`) unless necessary. Subtle bugs...

Reply:
got it!